actioncable/lib/action_cable/channel/base.rb

```diff
@@ -247,6 +247,7 @@ def processable_action?(action)
         end
 
         def dispatch_action(action, data)
+          return if subscription_rejected?
```

Comment:
This is arguably a protocol error on the part of the client; seems worth a log message... "ignoring [action signature] on rejected subscription" or something.